### PR TITLE
Notebooks with models runner

### PR DESCRIPTION
**Pull request by vifraa**
Added DataSample that can take x and y.
Fixed error in ModelsRunner where it could not be imported with data other than the hardcoded one.
Converted notebooks to use ModelsRunner.

Some cleanup in MLModels code.

### CODE REVIEW

File: models/4tt_models.ipynb [hunk 1]
```diff
@@ -14,7 +14,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 11,
```
Comment (carlostling) [inline models/4tt_models.ipynb:17]: Not sure what this does, but 1 -> 11 is intentional i assume

Comment (carlostling) [reply models/4tt_models.ipynb:17]: nvm it has to do with how many times the cell was run

File: models/MLmodels/ModelsRunner.py [hunk 2]
```diff
@@ -13,21 +13,23 @@
...
-        'Ridge': Ridge(),
-        'ElasticNet': ElasticNet(),
-        'DecisionTree': DecisionTree(),
-        'RandomForest': RandomForest(),
-        'NeuralNetwork': NeuralNetwork()
-    }
-
-    def __init__(self, path=None):
+    def __init__(self, path=None, data_sample=None):
         if path is not None:
```
Comment (carlostling) [inline models/MLmodels/ModelsRunner.py:17]: tip: this can be written as "if not path", in python "None" equivalates to "False" in conditions

**Review by carlostling — APPROVED** [review]
see line-specific comments, nothing to change. 
# APPROVED
## finicky stuff that doesnt matter
snake_case not used everywhere

**Review by henriklagergren — APPROVED** [review]
Looking good to me (I'm not a python expert like Carl)! ;)

**Pull request merged**